### PR TITLE
fixed a camera timeout issue occurring when the flat exposure time is…

### PR DESCRIPTION
**Pull request by decarlof**
… much shorted than the projection exposure time

### CODE REVIEW

**Comment by decarlof** [comment]
This is touching tomoscan.py

**Comment by MarkRivers** [comment]
Can you create an issue explaining the problem and how this solves it?  I am not sure I understand.

File: tomoscan/tomoscan.py [hunk 1]
```diff
@@ -534,7 +534,7 @@ def set_exposure_time(self, exposure_time=None):
                 exposure_time = self.epics_pvs['ExposureTime'].value
             self.epics_pvs['CamAcquireTime'].put(exposure_time, wait=True, timeout = 10.0)
 
-    def set_flat_exposure_time(self, exposure_time=None):
+    def set_scan_exposure_time(self, exposure_time=None):
         """Sets the camera exposure time for flat fields.
```
Comment (nikitinvv) [inline tomoscan/tomoscan.py]: @decarlof please update the description

File: tomoscan/tomoscan_pso.py [hunk 2]
```diff
@@ -291,6 +291,7 @@ def compute_positions_PSO(self):
         # Compute the time for each frame
         time_per_angle = self.compute_frame_time()
         self.motor_speed = np.abs(self.rotation_step) / time_per_angle
+        print(f'{self.rotation_step=}, {time_per_angle=}, {self.motor_speed=}')
```
Comment (nikitinvv) [inline tomoscan/tomoscan_pso.py]: @decarlof remove this

File: tomoscan/tomoscan.py (hunk 1 above)
Comment (decarlof) [reply tomoscan/tomoscan.py]: addressed [here](https://github.com/tomography/tomoscan/pull/144/commits/76af9722ed7c6a9be60904c7fc3d392720b38e2f)

File: tomoscan/tomoscan_pso.py (hunk 2 above)
Comment (decarlof) [reply tomoscan/tomoscan_pso.py]: addressed [here](https://github.com/tomography/tomoscan/pull/144/commits/76af9722ed7c6a9be60904c7fc3d392720b38e2f)

**Review by MarkRivers — COMMENTED** [review]
This looks fine to me.

**Review by MarkRivers — APPROVED** [review]

**Pull request merged**